LESS/graphics.less

```diff
@@ -0,0 +1,24 @@
+.banner-wrapper{
+    max-width: 100%;
+    overflow: hidden;
+    text-indent: -10%;
+}
+
+.banner-img{
+    max-width: 115%;
+}
+
+
+@media(min-width: 600px) {
```

Comment:
Rather than declaring the media query in its own line, you can just put it straight into banner wrapper like this
```
.banner-wrapper {
max-width: 100%;
overflow: 
text-indent: -10%;
@media(min-width: 600px) {
    display: flex;
    justify-content: center;
  }
}
```